docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
+=============
+
+.NET is a very flexible and naturally cross-platform [specification](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) for building developer products. It is used for all of the most popular app categories: desktop, mobile, cloud, gaming and IoT. 
+
+Major commercial and open source .NET products are listed below, in order of introduction.
```

Comment:
Is this the best way to introduce them?  As a new user I wouldn't really care about historical context unless this were specifically a "timeline" document of sorts.  Instead, I would probably expect to see an ordering like: `.NET Core --> .NET Framework --> Mono --> Unity --> .NET Native --> Windows IoT Core`.  I would think that this ordering would most reflect interest based on emphasis we've been placing on products and total number of users.